libdnf/utils/CompressedFile.cpp

```diff
@@ -0,0 +1,59 @@
+#include "CompressedFile.hpp"
+#include <utility>
+#include <sstream>
+extern "C" {
+#   include <solv/solv_xfopen.h>
+};
+
+libdnf::CompressedFile::CompressedFile(std::string filePath) : File(std::move(filePath)) {}
```

Comment:
1. Using `std::move(filePath)` is pointless until you don't have `File(str::string && filerPath)` constructor.
Then you can do:
copy variant:
libdnf::CompressedFile::CompressedFile(const std::string & filePath) : File(filePath) {}
move variant:
libdnf::CompressedFile::CompressedFile(std::string && filePath) : File(std::move(filePath)) {}